wof/WaterML_1_1.py

```diff
@@ -6,7 +6,7 @@
 # Generated Thu Oct 01 11:55:06 2015 by generateDS.py version 2.17a.
 #
 # Command line options:
-#   ('-o', 'wof\\WaterML_1_1.py')
+#   (u'-o', u'wof\\WaterML_1_1.py')
```

Comment:
It is a gigantic diff and GitHub even refuses to render :smile: 
But there are some noise from automate replaces. Just ignore that.